app/assets/stylesheets/application.css.scss

```diff
@@ -15,3 +15,123 @@
  *= require foundation_and_overrides
 
  */
+
+ body
+ {
+   background: {
+     image: asset-url("concert.jpg");
+   }
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-size: cover;
+ }
+
+.newest-shows
+{
+  width: 30%;
+  background: {
+    color: #170563;
+  }
+  opacity: .94;
+  margin-left: 3%;
+  float: left;
+}
+
+.hot-shows
+{
+  width: 30%;
+  background: {
+    color: #170563;
+  }
+  margin-left: 2%;
+  opacity: .94;
+  float: left;
+}
+
+.upcoming-shows
+{
+  width: 30%;
+  background: {
+    color: #170563;
+  }
+  margin-left: 2%;
+  opacity: .94;
+  float: left;
+}
+
+.page-content
+{
+  width: 90%;
+  margin: 8% auto;
+}
+
+.front-search {
+  display: inline-block;
+  width: 50%;
+  margin-top: 2%;
+}
+
+input#front-search-bar {
+  float: left;
+  width: 60%;
+  margin-left: 100px;
+}
+
+input#front-search-button {
+  float: left;
+  position: relative;
+  transform: translateY(-2%);
+  text-align: center;
+}
+
+.search-section {
+  text-align: center;
+}
+
+.show-details {
+  margin: 1%;
+}
+
+.band-and-venue {
+  text-align: center;
+}
+
+.band-name {
+  text-align: center;
+}
+
+.venue-name {
+  text-align: center;
+}
+
+.date-time {
+  text-align: center;
+}
+
+.votes {
+  float: left;
+  text-align: center;
+}
+
+.ticket-url {
+  float: right;
+}
+
+.header {
+  color: white;
+}
+
+.newest-header {
+  text-align: center;
+  color: white !important;
+}
+
+.hot-header {
+  text-align: center;
+  color: white !important;
+}
+
+.upcoming-header {
+  text-align: center;
+  color: white !important;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used